bobo.py

```diff
@@ -48,8 +49,10 @@ def main():
     lucky, date = get_lucky()
     send_msg(f"<b>Szczesliwy numerek: {get_name(str(lucky))} ({lucky})</b>")
 
-    send_msg(f"<i>Dzisiaj będzie {len(get_lessons())} lekcji</i>")
-
+    start_hour = get_lesson_start()
+    end_hour = get_lesson_end    
+    send_msg("Dzisiaj lekcje trwają od " + (LESSON_HOURS[start_hour]["start"]) + " do " + (LESSON_HOURS[end_hour]["end"])
```

Comment:
You forgot to close the braces :wink: 
```suggestion
    send_msg("Dzisiaj lekcje trwają od " + (LESSON_HOURS[start_hour]["start"]) + " do " + (LESSON_HOURS[end_hour]["end"]))
```